GVRf/Framework/framework/src/main/jni/objects/scene.cpp

```diff
@@ -23,9 +23,15 @@
 #include "objects/scene_object.h"
 
 namespace gvr {
+
+Scene* Scene::main_scene_ = NULL;
```

Comment:
Could this writable static be a concern in an environment where the user has multiple mainScenes and adds sceneObjects with colliders to some of them that not necessarily are in the active mainScene?

There could be subtle errors. For example the first time the app launches the ctor below will set the main scene but in subsequent app starts this may not be the case as the process frequently lingers on after app exit. It is all covered seemingly by the GVRViewManager but then why have this in the ctor?

If possible do not use writable static data. In general, what do you think about having fixed one and only mainScene?